sceptre/config/reader.py

```diff
@@ -552,6 +552,11 @@ def _construct_stack(self, rel_path, stack_group_config=None):
                 )
 
         s3_details = self._collect_s3_details(stack_name, config)
+        disable_rollback = (
+            True
+            if self.context.command_params.get("disable_rollback")
+            else config.get("disable_rollback", False)
+        )
```

Comment:
```suggestion
        # If disable/enable rollback was specified on the command line, use that. Otherwise,
        # fall back to the stack config.
        disable_rollback = self.context.command_params.get("disable_rollback")
        disable_rollback = config.get("disable_rollback", False) if disable_rollback is None else disable_rollback
```

Reply:
Just realized this might be cleaner:
```python
        disable_rollback = self.context.command_params.get("disable_rollback")
        if disable_rollback is None:
          disable_rollback = config.get("disable_rollback", False)
```